altair_saver/savers/_node.py

```diff
@@ -34,33 +35,46 @@ def exec_path(name: str) -> str:
     raise ExecutableNotFound(name)
 
 
-def vl2vg(spec: JSONDict) -> JSONDict:
+def vl2vg(spec: JSONDict, vega_cli_options: Optional[List] = None) -> JSONDict:
     """Compile a Vega-Lite spec into a Vega spec."""
     vl2vg = exec_path("vl2vg")
     vl_json = json.dumps(spec).encode()
-    vg_json = check_output_with_stderr([vl2vg], input=vl_json)
+    cmd = [vl2vg]
+    if vega_cli_options:
+        cmd += vega_cli_options
+    vg_json = check_output_with_stderr(cmd, input=vl_json)
     return json.loads(vg_json)
 
 
-def vg2png(spec: JSONDict) -> bytes:
+def vg2png(spec: JSONDict, vega_cli_options: Optional[List] = None) -> bytes:
     """Generate a PNG image from a Vega spec."""
     vg2png = exec_path("vg2png")
     vg_json = json.dumps(spec).encode()
-    return check_output_with_stderr([vg2png], input=vg_json)
+    cmd = [vg2png]
+    if vega_cli_options:
+        cmd += vega_cli_options
+    return check_output_with_stderr(cmd, input=vg_json)
```

Comment:
I'd make this a bit more concise:
```
return check_output_with_stderr([vg2png, *(vega_cli_options or [])], input=vg_json)
```
Similar in the other two functions

Reply:
Done.